docs/How_to_Guides/authentication.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: Authentication
+sidebar_position: 1
+---
+
+# Authentication
+
+In order to push or pull artifacts or images to an OCI registry, we need to prove our identity using authentication. Following this, authorization provides the required permissions to the user.
+
+In ORAS, we can use [`oras login`](../commands/oras_login.mdx) in advance to authenticate ourselves for private registries. By default, this command stores the credentials in `~/.docker/config.json` (same file used by the docker client). If you have previously authenticated to a registry using `docker login`, the credentials will be reused.
```

Comment:
Please note that this authentication approach is easy but not secure. The credential helper is the recommended way. There will be warning messages if no credential helper is configured in the user's machine. You can borrow some content from https://notaryproject.dev/docs/how-to/registry-authentication/